src/main/java/pl/zimi/zadanka/Zad7.java

```diff
@@ -0,0 +1,113 @@
+package pl.zimi.zadanka;
+
+/*
+    Source: https://leetcode.com/problems/maximum-depth-of-binary-tree/
+ */
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Definition for a binary tree node.
+ * public class TreeNode {
+ * int val;
+ * TreeNode left;
+ * TreeNode right;
+ * TreeNode() {}
+ * TreeNode(int val) { this.val = val; }
+ * TreeNode(int val, TreeNode left, TreeNode right) {
+ * this.val = val;
+ * this.left = left;
+ * this.right = right;
+ * }
+ * }
+ */
+public class Zad7 {
+
+    public static int maxDepth(TreeNode root) {
+
+        return exploreTree(root, 0, 0);
+    }
+
+    public static int exploreTree(TreeNode node, int depth, int maxDepth) {
+        depth++;
+        if (node == null) return maxDepth;
+        if (node.left != null) maxDepth = exploreTree(node.left, depth, maxDepth);
+        if (node.right != null) maxDepth = exploreTree(node.right, depth, maxDepth);
+        if (depth > maxDepth) maxDepth = depth;
+        return maxDepth;
+    }
+
+    public static int intelijMaxDepth(Integer[] inputArray) {
```

Comment:
to przenieśliśmy do testów... zrobiłeś commit i push?

Reply:
nie wrzuciłem  commita. teraz jest poprawione